package.json

```diff
@@ -11,7 +11,10 @@
     "bootstrap": "lerna bootstrap --hoist",
     "pretest": "lerna run --scope demo build",
     "test": "mocha test/*.js",
-    "lint": "eslint ./packages --ext .js"
+    "lint": "eslint ./packages --ext .js",
+    "start": "lerna run start --scope demo",
```

Comment:
Let's add the `--stream` flag to each of these `lerna run` commands, because they are ongoing processes. Otherwise we don't see the output of these processes in the terminal.